go.mod

```diff
@@ -76,6 +75,7 @@ require (
 	github.com/xrash/smetrics v0.0.0-20201216005158-039620a65673 // indirect
 	go.opencensus.io v0.24.0 // indirect
 	go.uber.org/mock v0.3.0 // indirect
+	golang.org/x/crypto v0.17.0 // indirect
```

Comment:
Why has it moved to the indirect dependencies?

Reply:
I updated it with `go get` and did not run `go mod tidy`